app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -171,24 +172,40 @@ Template.visitorInfo.events({
 	'click .close-livechat'(event) {
 		event.preventDefault();
 
-		modal.open({
-			title: t('Closing_chat'),
-			type: 'input',
-			inputPlaceholder: t('Please_add_a_comment'),
-			showCancelButton: true,
-			closeOnConfirm: false,
-		}, (inputValue) => {
-			if (!inputValue) {
-				modal.showInputError(t('Please_add_a_comment_to_close_the_room'));
-				return false;
-			}
+		if (settings.get('Livechat_conversation_finished_message_enabled')) {
```

Comment:
Please, add a const (arrow function) before this line, and then you can just do something like this:
```
const closeRoom = (comment) => ...here you can call Meteor.call('livechat:closeRoom' ....
if (!settings.get('Livechat_conversation_finished_message_enabled')) {
    return closeRoom(...);
}
... here you can call de modal... closeRoom... and so on..
```